rero_ils/modules/items/api_views.py

```diff
@@ -103,6 +103,13 @@ def decorated_view(*args, **kwargs):
                 'action_applied': action_applied
             })
         except CirculationException as error:
+            patron = False
+            # Detect patron details
+            if data.get('patron_pid'):
+                patron = Patron.get_record_by_pid(data.get('patron_pid'))
+            # Add more info in case of blocked patron (for UI)
+            if patron and patron.get('blocked', {}) is True:
+                abort(403, "BLOCKED USER")
```

Comment:
For me it's not a good idea to derivate the CirculationException to know if Patron is blocked. It will be better to send a signal to request at registered function if some action are available. Each modules could answers with their own reasons. CircPolicies could answer `YES`, and Patrons could answer `NO` because "xxxx". The main function should be able to merge answers and send a coherent response.

Reply:
We could probably check that this week. I do things to avoid modifying a lot backend and UI. And I'm not comfortable with UI about fetching backend errors details.

Reply:
Info : @zannkukai and me talked about that. We can improve this behaviour later in [US300]( https://tree.taiga.io/project/rero21-reroils/us/300?no-milestone=1).